include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -3882,7 +3882,7 @@ _RandomAccessIterator
 __brick_min_element(_RandomAccessIterator __first, _RandomAccessIterator __last, _Compare __comp,
                     /* __is_vector = */ ::std::true_type) noexcept
 {
-#if (_PSTL_UDR_PRESENT || _ONEDPL_UDR_PRESENT)
```

Comment:
Just my opinion, but I prefer. 
`#if _ONEDPL_UDR_PRESENT  //_PSTL_UDR_PRESENT`

The `||` is our creation, I think upstream is just `#if _PSTL_UDR_PRESENT`